### ISSUE TITLE
Build Plugins have different situation in TRT6 & TRT7

### ISSUE THREAD

**Issue by tingggggg** [NVIDIA/TensorRT#706]
## Description

I build env refer (https://github.com/rmccorm4/tensorrt-utils/tree/master/OSS) in TRT6 & TRT7

Both of them be build successfull.

But when I print out the total custom plugins, I found not same that in TRT6 & TRT7

TRT6

> RnRes2Br1Br2c_TRT
CgPersistentLSTMPlugin_TRT
RnRes2Br2bBr2c_TRT
SingleStepLSTMPlugin
FancyActivation
ResizeNearest
Split
InstanceNormalization
GridAnchor_TRT
NMS_TRT
Reorg_TRT
Region_TRT
Clip_TRT
LReLU_TRT
PriorBox_TRT
Normalize_TRT
RPROI_TRT
BatchedNMS_TRT
FlattenConcat_TRT

TRT7

> GroupNormalizationPlugin
> CustomEmbLayerNormPluginDynamic
> CustomFCPluginDynamic
> CustomGeluPluginDynamic
> CustomQKVToContextPluginDynamic
> CustomSkipLayerNormPluginDynamic
> BatchTilePlugin_TRT
> BatchedNMS_TRT
> CoordConvAC
> CropAndResize
> DetectionLayer_TRT
> FlattenConcat_TRT
> GenerateDetection_TRT
> GridAnchor_TRT
> GridAnchorRect_TRT
> InstanceNormalization_TRT
> MultilevelCropAndResize_TRT
> MultilevelProposeROI_TRT
> NMS_TRT
> Normalize_TRT
> PriorBox_TRT
> ProposalLayer_TRT
> Proposal
> PyramidROIAlign_TRT
> Region_TRT
> Reorg_TRT
> ResizeNearest_TRT
> RPROI_TRT
> SpecialSlice_TRT

I need to infer faster & mask RCNN & Retinanet, so maybe I need `PyramidROIAlign_TRT`, `ResizeNearest_TRT` and `CropAndResize`.

I do infer maskRCNN and Retinanet is successful in TRT7, but not in TRT6.

Any one face same problem!?
I think maybe some different in CMakefile of branch TensorRT 6 and branch TensorRT 7??

## Environment

**TensorRT Version**: 6 & 7
**GPU Type**: 
**Nvidia Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System + Version**: 
**Python Version (if applicable)**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**:

**Comment by rajeevsrao** [comment]
@tingggggg you should be able to leverage all our TensorRT 7.x plugins in TensorRT 6.1 since the recommended PluginV2 interfaces (`IPluginV2DynamicExt`, `IPluginV2IOExt`) haven't been updated since then.

**Comment by ttyio** [comment]
Close since no activity for more than 3 weeks, please reopen if you still have question, thanks
